starfinder/db/models.py

```diff
@@ -281,6 +383,23 @@ class Descriptor(db_engine.Model, ModelBase, HasId):
 
     spells = orm.relationship('SpellDescriptor', backref='descriptor')
 
+class Language(db_engine.Model, ModelBase, HasId):
+    name = db_engine.Column(db_engine.String(64), nullable=False)
+    world_id = db_engine.Column(db_engine.ForeignKey("worlds.id"),
+                                     nullable=False)
+    race_id = db_engine.Column(db_engine.ForeignKey("races.id"),
+                                     nullable=False)
+    other = db_engine.Column(db_engine.String(64), nullable=False)
+
+class MagicSchool(db_engine.Model, ModelBase, HasId):
+    id = db_engine.Column(db_engine.Integer(), primary_key=True)
+    name = db_engine.Column(db_engine.String(64), nullable=False)
+    description = db_engine.Column(db_engine.String(64), nullable=False)
+
+    spells = orm.relationship('Spell', backref='magicschool')
```

Comment:
this one's sneaky, but be sure to put an underscore wherever two words are combined in lowercase. in this case, `magic_school`